src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -290,7 +300,8 @@
 
     // Define Connector configuration group
     config.define(FLUSH_SIZE_CONFIG, Type.INT, Importance.HIGH, FLUSH_SIZE_DOC, CONNECTOR_GROUP, 1, Width.SHORT, FLUSH_SIZE_DISPLAY)
-        .define(ROTATE_INTERVAL_MS_CONFIG, Type.LONG, ROTATE_INTERVAL_MS_DEFAULT,Importance.HIGH, ROTATE_INTERVAL_MS_DOC, CONNECTOR_GROUP, 2, Width.SHORT, ROTATE_INTERVAL_MS_DISPLAY)
+        .define(ROTATE_INTERVAL_MS_CONFIG, Type.LONG, ROTATE_INTERVAL_MS_DEFAULT, Importance.HIGH, ROTATE_INTERVAL_MS_DOC, CONNECTOR_GROUP, 2, Width.SHORT, ROTATE_INTERVAL_MS_DISPLAY)
+        .define(ROTATE_SCHEDULE_MS_CONFIG, Type.LONG, ROTATE_SCHEDULE_MS_DEFAULT, Importance.HIGH, ROTATE_SCHEDULE_MS_DOC, CONNECTOR_GROUP, 2, Width.SHORT, ROTATE_SCHEDULE_MS_DISPLAY)
```

Comment:
The order in group needs to be adjusted.  This is used to order the configs inside a group. Also, should the importance of this config to be high? I think we can use medium and the reasoning is that the connector still function properly if this config is not provided.